### PR TITLE
add link to sheets page

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

Add sheets card to labs page

**Screenshots**
Current:
image

Figma:
image

### CODE REVIEW

File: src/assets/sheets_card.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...L64.485 510.619a8 8 0 0 0 0 11.313l267.292 267.292a8 8 0 0 0 11.314 0L615.225 517.09a8 8 0 0 0 0-11.313L347.933 238.485a8 8 0 0 0-11.314 0zM63.071 509.205c-3.905 3.905-3.905 10.236 0 14.142l267.292 267.291c3.905 3.906 10.237 3.906 14.142 0l272.134-272.133c3.905-3.906 3.905-10.237 0-14.142L349.347 237.071c-3.905-3.906-10.237-3.906-14.142 0L63.071 509.205zM425.647-133.212L262.893 29.542a8 8 0 0 0 0 11.314l139.09 139.089a8 8 0 0 0 11.313 0L576.05 17.191a8 8 0 0 0 0-11.313l-139.089-139.09a8 8 0 0 0-11.314 0zM261.479 28.128c-3.905 3.905-3.905 10.237 0 14.142l139.089 139.089c3.906 3.905 10.237 3.905 14.143 0L577.464 18.606c3.905-3.906 3.905-10.237 0-14.142l-139.089-139.09c-3.905-3.905-10.237-3.905-14.142 0L261.479 28.128z"/><path d="M37.475 185.485l-197.99 197.99a8 8 0 0 0 0 11.314L33.887 589.191a8 8 0 0 0 11.314 0l197.99-197.99a8 8 0 0 0 0-11.314L48.789 185.485a8 8 0 0 0-11.314 0zm-199.404 196.576c-3.905 3.905-3.905 10.237 0 14.142L32.473 590.605c3.905 3.905 10.237 3.905 14.142 0l197.99-197.99c3.905-3.905 3.905-10.237 0-14.142L50.203 184.071c-3.905-3.905-10.237-3.905-14.142 0l-197.99 197.99z"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h558v300H0z"/></clipPath></defs></svg>
```
Comment (Partysun) [inline src/assets/sheets_card.svg]: Why do we have this, if it is the same as for Sangraphs's card. @GarageInc

Comment (haritonasty) [reply src/assets/sheets_card.svg]: yes, just use background from `dashboards` card from figma

Comment (GarageInc) [reply src/assets/sheets_card.svg]: Fixed

File: src/App.js [hunk 2]
```diff
@@ -316,40 +343,15 @@ export const App = ({
...
-          exact
-          path='/support'
-          render={props => (
-            <ExternalRedirect to={'mailto:info@santiment.net'} />
-          )}
-        />
+        {ExternalRoutes.map(links => {
+          return links.routes.map(name => (
+            <Route
+              key={name}
```
Comment (Partysun) [inline src/App.js]: two components will be with the same KEY "docs". @GarageInc

File: src/App.js [hunk 3]
```diff
@@ -155,7 +155,7 @@ const ExternalRoutes = [
   },
   {
     to: 'https://docs.santiment.net',
-    routes: ['docs', 'apiexamples']
+    routes: ['apiexamples', 'apiexamples']
```
Comment (Partysun) [inline src/App.js]: ['apiexamples']

**Pull request merged**